src/tinkers/resources/META-INF/mods.toml

```diff
@@ -15,15 +15,21 @@ description = '''
     An addon for Ex Nihilo: Sequentia that adds support for Tinkers Construct.
     '''
 [[dependencies.exnihilotinkers]]
+modId = "forge"
+mandatory = true
+versionRange = "${forge_version}"
+ordering = "NONE"
+side = "BOTH"
+[[dependencies.exnihilotinkers]]
 modId = "exnihilosequentia"
 mandatory = true
 versionRange = "${minecraft_version_range}"
-ordering = "AFTER"
+ordering = "NONE"
```

Comment:
I need to do this in other addons. Since getting rid of those enums this doesn't matter. Great catch